lib/rspec/matchers/built_in/captures.rb

```diff
@@ -0,0 +1,78 @@
+module RSpec
+  module Matchers
+    module BuiltIn
+      # @api private
+      # Provides the regex with captures matcher
+      class Captures < BaseMatcher
```

Comment:
I'm not usually one for inheritance, but is this a legit case for Captures to inherit from Match?